docs/subnets/deploying-subnets-on-prod.md

```diff
@@ -0,0 +1,191 @@
+
+#### Database Copy
+
+Good way to cut down on bootstrap times on multiple nodes is database copy. Database is identical across nodes, and as such can safely be copied from one node to another. Just make sure to that the node is not running during the copy process, as that can result in a corrupted database. Database copy procedure is explained in detail [here](../nodes/maintain/node-backup-and-restore.md#database).
+
+Please make sure you don't reuse any node's node-id by accident, especially don't restore another node's ID, see [here](../nodes/maintain/node-backup-and-restore.md#nodeid) for details. Each node must has its own unique node id, otherwise, the nodes sharing the same id will not behave correctly, which will impact your validator's uptime, thus staking rewards, and the stability of your Subnet.
+
+## Subnet Deploy
+
+Once you have the nodes set up you are ready to deploy the actual Subnet. Right now, the recommended tool to do that is [Subnet-CLI](https://github.com/ava-labs/subnet-cli).
+
+Instructions for using the Subnet-CLI can be found [here](../subnets/subnet-cli.md). We will highlight the main steps below.
+
+### Ledger HW Wallet
+
+When creating the Subnet, you will be required to have a private key that will control the administrative functions of the Subnet (adding validators, managing the configuration). Needless to say, whoever has this private key has complete control over the Subnet and the way it runs. Therefore, protecting that key is of the utmost operational importance. Which is why we strongly recommend using a hardware wallet such as a [Ledger HW Wallet](https://www.ledger.com/) to store and access that private key. 
+
+Of course, Subnet-CLI supports the usage of a Ledger HW wallet. Take advantage of that because losing control of the managing would be catastrophic. General instruction on how to use a Ledger device with Avalanche can be found [here](https://support.avax.network/en/articles/6150237-how-to-use-a-ledger-nano-s-or-nano-x-with-avalanche).
+
+### Genesis File
+
+The structure that defines the most important parameters in a Subnet is found in the genesis file, which is a `json` formatted, human-readable file. Describing the contents and the options available in the genesis file is beyond the scope of this document, and if you're ready to deploy your Subnet to production you probably have it mapped out already. 
+
+If you want to review, we have a description of the genesis file in our document on [customizing EVM Subnets](customize-a-subnet.md).
+
+### Subnet-CLI Wizard
+
+Creating a Subnet is a multistep process. You need to:
+* create a `WMID`
```

Comment:
[MD032/blanks-around-lists: Lists should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.25.1/doc/Rules.md#md032---lists-should-be-surrounded-by-blank-lines)